scripts/performance/run_script.py

```diff
@@ -49,16 +49,37 @@ def main():
     elif args.model_name == "llama31" and args.model_size == "405b":
         recipe = llama31_405b_pretrain_config(mock=True, precision_config=precision_config)
     elif args.model_name == "deepseek" and args.model_size == "v3":
+        enable_deepep = bool(args.gpu.lower() in ["h100"])
+        A2A_1F1B = bool(args.gpu.lower() in ["h100"])
+
+        pp, vp = (8, 4) if args.gpu.lower() in ["h100"] else (4, 8)
         recipe = deepseek_v3_pretrain_config(
             mock=True,
             precision_config=precision_config,
             # NOTE: IMPORTANT: PLEASE SET PP-VP size here to correctly set the pp-vp layout
-            pipeline_parallelism=4,
-            virtual_pipeline_parallelism=1,
+            pipeline_parallelism=pp,
+            virtual_pipeline_parallelism=vp,
+            enable_deepep=enable_deepep,
         )
         from megatron.bridge.training.utils.moe_token_drop import apply_moe_token_drop
 
-        recipe.model = apply_moe_token_drop(recipe.model)
+        if enable_deepep:
+            recipe.model.moe_router_force_load_balancing = True
+        else:
+            recipe.model = apply_moe_token_drop(recipe.model)
```

Comment:
We should add a knob for token drop/dropless, where it defaults to dropless.
When we measure the perf of B200 and GB200, we should be using `tokendrop + enable_deepep:False` and `tokendropless + enable_deepep:True` for H100.